src/mlpack/methods/linear_svm/linear_svm_impl.hpp

```diff
@@ -48,10 +48,35 @@ LinearSVM<MatType>::LinearSVM(
     delta(delta),
     fitIntercept(fitIntercept)
 {
+  if (numClasses == 1)
```

Comment:
Shouldn't this be `<= 1` not `== 1`?  Further, we should put the check in `Train()`, since it is valid to create a `LinearSVM` object without training it.

Reply:
Resolved and added this part in `Train()` also.